uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_variables.scss

```diff
@@ -1,7 +1,8 @@
 // Variables for Sidebar
-$light-gray : #D2D2D2;
-$secondary-colour: #707C88;
-$sidebar-main-bg: #fff;
+$light-blue:        #41AFCB;
+$white:             #FFFFFF;
+$secondary-colour:  #707C88;
+$sidebar-main-bg:   #fff;
```

Comment:
This is not necessary, as you already have $white.

Reply:
Thanks for catching! fixed.